include/nw4r/ut/Rect.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <Common.h>
+
+namespace nw4r {
+namespace ut {
+
+struct Rect {
```

Comment:
If you care about finishing the implementation of `Rect`, OGWS has the following inlines:
- `SetWidth`, `SetHeight`
- `Normalize`
- `MoveTo`

Reply:
In the debug build, BBA emits:

- `Normalize()`
- `Rect(f32, f32, f32, f32)`
- `operator=(const Rect &rhs)`
- `GetHeight() const`
- `GetWidth() const`
- `Rect()`
- `MoveTo(f32, f32)`
- `Rect(const Rect &rhs)`

On release, only the default constructor, destructor, and `GetWidth()` are emitted.

Reply:
Maybe it just doesn't have them then. I only remember needing them in OGWS for TextWriterBase.